src/qibotn/backends/qmatchatea.py

```diff
@@ -23,6 +23,9 @@ def __init__(self):
         self.name = "qibotn"
         self.platform = "qmatchatea"
 
+        # Default precision
+        self.precision = "double"
```

Comment:
base `qibo` does not use `precision` anymore since https://github.com/qiboteam/qibo/pull/1625 because it was redundant with `dtype`